src/cdk/v2/destinations/bluecore/data/bluecoreTrackConfig.json

```diff
@@ -0,0 +1,77 @@
+[
```

Comment:
There seems to be some common fields between identify and track config.Can we move those out to commonConfig

Reply:
yeah, will update this